### PR TITLE
feat(cors): Add support for Cross Origin Requests

### PR DESCRIPTION
**Pull request by soapdog**
Changed page module to add support for cross origin requests via `page:enable_cors()`. Added documentation and some tests as well.

### CODE REVIEW

File: src/sailor/page.lua [hunk 1]
```diff
@@ -145,6 +145,51 @@ function Page:json(data, args)
...
+--
+--  Access-Control-Allow-Origin = "*"
+--
+-- which should be enough for most cases.
+--
+--
+function Page:enable_cors(data)
+    local config = {
+        allow_origin = data.allow_origin or "*",
+        expose_headers = data.expose_headers or nil
```
Comment (Etiene) [inline src/sailor/page.lua]: missing coma? :)

Comment (soapdog) [inline src/sailor/page.lua]: missing commas... damn, and I tested this before. Argh, sorry. Fixing...

**Comment by soapdog** [comment]
fixing these new errors...

**Comment by soapdog** [comment]
@Etiene sorry for the crazy pushing, I was having trouble running the tests locally. Now, it is working. I fixed the code and it now passes `sailor test`. Not sure about the coveralls thing. Can it be merged?

**Comment by Etiene** [comment]
awesome stuff!!! :D thanks for the great pr! 
yea, the coverall decreased because probably this new method needs automated tests, but Im not sure how that could be done. what do you think?

**Comment by soapdog** [comment]
I am new to Lua testing, so I have no idea how it could be done. I added two naive tests but I would like to see better ones.

**Pull request merged**